src/data/ha-state-history-data.html

```diff
@@ -131,20 +131,13 @@
       };
     }
 
+    // debounce the observer because attribute changes are sequential.
```

Comment:
I don't think that this comment is correct. Property changes are no longer sequential in Polymer 2. It could be that certain things get loaded very closely after one another making it look like it's sequential